packages/nextjs/services/database/seed.data.example.ts

```diff
@@ -140,13 +140,23 @@ export const seedChallenges: (typeof challenges.$inferInsert)[] = [
     previewImage: "/assets/challenges/stablecoins.svg",
   },
   {
-    id: ChallengeId.DEPLOY_TO_L2,
-    challengeName: "⚡ Deploy to Layer 2",
+    id: ChallengeId.ZK_VOTING,
+    challengeName: "🔒 ZK Voting",
     autograding: false,
     description:
-      "🚀 Ethereum L2s make blockchain apps fast and cheap, bringing us closer to mainstream adoption! Most L2s are EVM compatible, meaning your app should work seamlessly across them with little to no changes—just deploy and go! In this challenge, you will deploy an app across multiple chains, including Optimism, Base, and Arbitrum, and experience the snappy, low-cost transactions while exploring how they make building scalable apps and games easier than ever.",
+      "🔒 Build your own decentralized voting system using zero-knowledge proofs. Let's write a contract that allows users to vote on a topic and the results will be computed off-chain using zero-knowledge proofs.",
     sortOrder: 8,
-    previewImage: "/assets/challenges/deployToL2.svg",
+    previewImage: "/assets/challenges/zkVoting.svg",
+  },
+  {
+    id: ChallengeId.ORACLES,
+    challengeName: "Oracles",
+    github: "scaffold-eth/se-2-challenges:challenge-oracles",
```

Comment:
remove github for now so in the `/start` page oracles challenge will have `coming soon` too

<img width="653" height="269" alt="Image" src="https://github.com/user-attachments/assets/aac7fcdd-048a-4fcd-a972-2c54c2537705" />